src/core/thread/key_manager.cpp

```diff
@@ -200,22 +209,96 @@ void KeyManager::Stop(void)
 }
 
 #if OPENTHREAD_MTD || OPENTHREAD_FTD
+
+Error KeyManager::StorePskc(void)
```

Comment:
Should we add this function only under `#if OPENTHREAD_CONFIG_PLATFORM_KEY_REFERENCES_ENABLE`?